packages/babel-helpers/src/helpers.js

```diff
@@ -425,6 +425,46 @@ helpers.inheritsLoose = defineHelper(`
   }
 `);
 
+// Based on https://github.com/WebReflection/babel-plugin-transform-builtin-classes
+helpers.wrapNativeSuper = defineHelper(`
+  var _gPO = Object.getPrototypeOf || function _gPO(o) { return o.__proto__ };
+  var _sPO = Object.setPrototypeOf || function _sPO(o, p) { o.__proto__ = p };
+  var _construct = Reflect.construct || function _construct(Parent, args, Class) {
```

Comment:
Shouldn't we check for Reflect first?

Reply:
agreed, and original plugin was doing it indeed:
https://github.com/WebReflection/babel-plugin-transform-builtin-classes/blob/master/src/index.js#L8

Reply:
I'll use `(typeof Reflect === "object" && Reflect.construct) || function ...` instead of the ternary (which is used in the original plugin) since someone might only polyfill *some* Reflect methods, so `Reflect.constructor` could be not defined.